src/GraphQL.Tests/Execution/VariablesTests.cs

```diff
@@ -109,7 +109,7 @@ public void executes_with_complex_input()
               fieldWithObjectInput(input: {a: ""foo"", b: [""bar""], c: ""baz""})
             }
             ";
-            var expected = "{ \"fieldWithObjectInput\": \"{\\\"a\\\":\\\"foo\\\",\\\"b\\\":[\\\"bar\\\"],\\\"c\\\":\\\"baz\\\"}\" }";
+            var expected = "{ \"fieldWithObjectInput\": \"{\\\"a\\\":\\\"foo\\\",\\\"b\\\":[\\\"bar\\\"],\\\"c\\\":\\\"baz\\\",\\\"d\\\":null}\" }";
```

Comment:
I don't think it is ... those are new tests I just added.  I will double-check though.